roundtripper.go

```diff
@@ -64,7 +64,7 @@ func (ctx *ProxyCtx) _roundTripWithLog(req *http.Request) (*http.Response, error
 		reqAndResp.captureContent = ctx.isLogWithContent
 
 		req := ctx.Req
-		if reqAndResp.captureContent && req.ContentLength > 0 {
```

Comment:
are you sure we can take out the `captureContent` condition ?  this is when a user asks for content or not..